ipni/storetheindex
# Support Ipni-Cid-Schema-Type HTTP header

The Ipni-Cid-Schema-Type HTTP header serves as an indication to advertisement publishers what type of data is being requested and is identified by the CID. This may help some publishers more quickly lookup the data.

- Remove `loadtest synthetic` command that was never used

The `synthetic` command is not used as there is not longer a way to import the generated data. This command was originally used to populate storetheindex with artificial data for testing. Test data is readily available from providers and CAR mirrors, and can be made available by a reference index provider, so there is not a need to generate data synthetically withing a storetheindex admin command.

Other changes:
- Update go-libp2p
- Update boxo
- Update README to remove references to comands that no longer exist

Closes #2662
